### PR TITLE
Add 4EVERLAND

### PR DESCRIPTION
**Pull request by 4everlandorg**
Add 4EVERLAND to web hosting, iaas, storage and media processing directories

## Requirements

 * [ ] This is Software as a Service not self hosted
 * [ ] It has a free tier not just a free trial
 * [ ] Pricing information is clearly visible without signup or phone calls
 * [ ] The submission mentions what is free
 * [ ] The submission is not already present in the list
 * [ ] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1153,6 +1153,7 @@ Update Time, five active automations, webhooks.
 
 ## Web Hosting
 
+  * [4EVERLAND](https://www.4everland.org/) — In-built development and deployment tools(e.g., visual and command line) to build, deploy and host decentralised websites/applications with consensus-driven Swarm network, free SSL, global CDN, and preview URLs. The gateway node includes dynamic DNS, Data l/0, and CertManager modules. Registered users can get free 6 GB IPFS storage, 300MB Arweve storage and 100 GB/month bandwidth.
```
Comment (ripienaar) [inline README.md:1156]: Thanks, we only list entries once so find a category that suits best.

Comment (4everlandorg) [reply README.md:1156]: Thank you for your reply, if so please include us under the Iaas category, is anything new i need to do?

Comment (ripienaar) [reply README.md:1156]: Just update your PR accordingly

Comment (4everlandorg) [reply README.md]: It's done, thanks.

**Comment by ripienaar** [comment]
thanks

**Pull request merged**